pokemonlib.py

```diff
@@ -80,7 +80,7 @@ def __init__(self):
         #logger.debug("Running pidof calcy got code %d: %s", return_code, stdout)
         #self.calcy_pid = stdout.decode('utf-8').strip()
         # cmd = ["adb", "-s", await self.get_device(), "logcat", "-T", "1", "-v", "brief", "--pid", self.calcy_pid]
-        cmd = ["adb", "-s", await self.get_device(), "logcat", "-T", "1", "-v", "brief" , "MainService:D i:D ClipboardReceiver:D *:S"]
+        cmd = ["adb", "-s", await self.get_device(), "logcat", "-T", "1", "-v", "brief" , "MainService:D j:D ClipboardReceiver:D *:S"]
```

Comment:
Any idea what does this character in particular mean?

Reply:
It is just a Tag aka a provider identification.
They should make it longer and more describing. I thing it was Calcy in the beginning.
They like hope nobody else in the very big Android dev community, will use the one letter j  :-O